bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -375,6 +375,8 @@ thing-type.netatmo.account.label = Netatmo Account
 thing-type.netatmo.account.description = This bridge represents an account, gateway to Netatmo API.
 thing-type.netatmo.doorbell.label = Smart Video Doorbell
 thing-type.netatmo.doorbell.description = The Netatmo Smart Video Doorbell device.
+thing-type.netatmo.tag.label = Smart Door Sensor
+thing-type.netatmo.tag.description = The Netatmo Smart Door or Window sensor.
```

Comment:
Please run the Maven command, this will add the new entries at the "right" place and add missing entries for your new channel type.

Reply:
Done